File: pubnub-gson/src/main/java/com/pubnub/api/endpoints/files/DeleteFile.java
package com.pubnub.api.endpoints.files;

import com.pubnub.api.endpoints.BuilderSteps;
import com.pubnub.api.endpoints.Endpoint;
import com.pubnub.api.endpoints.files.requiredparambuilder.FilesBuilderSteps;
import com.pubnub.api.models.consumer.files.PNDeleteFileResult;

public interface DeleteFile extends Endpoint<PNDeleteFileResult> {
    interface Builder extends BuilderSteps.ChannelStep<FilesBuilderSteps.FileNameStep<FilesBuilderSteps.FileIdStep<DeleteFile>>> {

    }
}

File: pubnub-kotlin/src/main/kotlin/com/pubnub/api/endpoints/files/DeleteFile.kt
package com.pubnub.api.endpoints.files

import com.pubnub.api.Endpoint
import com.pubnub.api.models.consumer.files.PNDeleteFileResult

/**
 * @see [PubNub.deleteFile]
 */
interface DeleteFile : Endpoint<PNDeleteFileResult?>
